crates/turborepo-lib/src/cli.rs

```diff
@@ -352,6 +352,11 @@ pub struct RunArgs {
     /// to identify which packages have changed.
     #[clap(long)]
     pub since: Option<String>,
+    /// Use "none" to remove prefixes from task logs. Note that using this
+    /// option means that tasks running in parallel may stream their logs
+    /// mixed together.
```

Comment:
Wordsmithing this is hard, but this almost makes it sound like only when using this option the logs may be mixed together. Maybe "Note that tasks running in parallel may interleave their logs and prefix is the only way to identify which task produced a log"